sdk/apis/tenancy/v1alpha1/types_workspace.go

```diff
@@ -168,7 +170,7 @@ type WorkspaceSpec struct {
 	//
 	// +optional
 	// +kubebuilder:validation:XValidation:rule="self == oldSelf",message="cluster is immutable"
-	Cluster string `json:"cluster,omitempty"`
+	Cluster logicalcluster.Name `json:"cluster,omitempty"`
```

Comment:
Isn't this a breaking change? We should add a release note in that case.

Reply:
true. Breaking in code.

Reply:
done